src/components/hooks/useDashboardAccess.js

```diff
@@ -1,4 +1,4 @@
-import shallow from 'zustand/shallow';
```

Comment:
Apologies Claudia, but I don't think this is right, because we import from `zustand/shallow` I think we _do_ want a default import not a named one here

Reply:
Resolved, this was my mistake, the above fix is correct